includes/wikia/services/RedirectService.class.php

```diff
@@ -82,11 +83,17 @@ public function getRedirects() {
 			$this->redirects = $this->prepareRedirects();
 		}
 
-		if ( isset( $this->pageType )
-			&& isset( $this->redirects[ $this->pageType ] )
-			&& is_array( $this->redirects[ $this->pageType ]) )
-		{
-			return $this->redirects[ $this->pageType ];
+
+		if ( isset( $this->pageType ) ) {
+			if (
+				isset( $this->redirects[ $this->pageType ] )
+				&& is_array( $this->redirects[ $this->pageType ])
```

Comment:
you first check if key is set and then you check if this is an array, why not other way around? If an array, then check the key.